dev_guide/templates.adoc

```diff
@@ -339,20 +339,31 @@ metadata:
   name: "cakephp-mysql-example" <1>
   annotations:
     openshift.io/display-name: "CakePHP MySQL Example" <2>
-    description: "An example CakePHP application with a MySQL database" <3>
+    description: "An example CakePHP application with a MySQL database.\n\nFor more information see https://github.com/openshift/cakephp-ex" <3>
```

Comment:
@jwforres Is the \n\n supposed to be in there? Just checking as I make follow-up edits. Thanks!